wordpress-to-github/index.js

```diff
@@ -33,13 +36,14 @@ const {
 const commitTitlePosts = "Wordpress Posts Update";
 const commitTitlePages = "Wordpress Pages Update";
 const commitTitleMedia = "Wordpress Media Update";
+const commitTitleApiRequests = "Wordpress API Requests Update";
```

Comment:
This is great!  Any way we could introduce a "name" or "commitTitle" to the config row for each API request to help ID the commit changes?

Reply:
The new code does add a unique label to the end of this string before writing out the commit. See line 523/524 of `wordpress-to-github/index.js`. 

This could certainly be improved. My goal here was just to re-use the same conventions as current code.